app/views/content/salaries-and-benefits.md

```diff
@@ -98,13 +99,13 @@ This extra support used to last 1 year, and teachers would be called 'newly qual
 
 ## Career progression
 
-There are different ways to progress and increase your salary.
+There are different ways to progress and increase your teacher salary.
```

Comment:
This one doesn't sound right. Might be a search term, but grammatically it should be teaching salary.